client/layout/index.jsx

```diff
@@ -543,7 +536,7 @@ export default withCurrentRoute(
 				userAllowedToHelpCenter,
 				currentRoute,
 				isGlobalSidebarVisible: shouldShowGlobalSidebar && ! sidebarIsHidden,
-				isGlobalSidebarCollapsed: shouldShowGlobalSiteSidebar && ! sidebarIsHidden,
+				isGlobalSidebarCollapsed: shouldShowGlobalSidebar && siteId && ! sidebarIsHidden,
```

Comment:
I don't know enough about the code changes to understand if this is intentional (maybe it's behind a feature flag?), but just noting that this looks kind of weird for me in the Reader - I don't see this on `trunk`, so I'm assuming it's related to this change. 

<img width="1030" alt="Screenshot 2024-04-25 at 11 13 10" src="https://github.com/Automattic/wp-calypso/assets/43215253/eaa5af4a-ef37-4367-a32f-1afe691143d9">

Reply:
Thanks, I'll take a look!

Reply:
> Edit: ah, sorry - just saw the to-do! My bad! 🙃

Jajaj, no problem :wink: